From 406274e9dcf87b4b1d522606bc089ebb01faa0fa Mon Sep 17 00:00:00 2001
From: Martin Stradiot <martin.stradiot@gmail.com>
Date: Sat, 7 Dec 2024 20:38:34 +0000
Subject: [PATCH] patch 1

---
 Makefile | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/Makefile b/Makefile
index 9f5730e..eb6ed25 100644
--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,5 @@
 
-#SUBDIRS =  misc-progs misc-modules \
-#           skull scull scullc scullp lddbus sculld scullv shortprint simple tty \
-#	   pci usb\
-#	   sbull snull short
-
-SUBDIRS =  misc-modules scull
+SUBDIRS = misc-modules scull
 
 all: subdirs
 
-- 
2.34.1

